src/py42/sdk/queries/fileevents/filters/activity_filter.py

```diff
@@ -0,0 +1,39 @@
+from py42.sdk.queries.fileevents.file_event_query import FileEventFilterStringField
+from py42.sdk.queries.query_filter import filter_attributes
+
+
+class TrustedActivity(FileEventFilterStringField):
+    """Class that filters events based on whether activity can be trusted.
+
+        Available event types are provided as class attributes:
+            - :attr:`FileCategory.INCLUDE`
+            - :attr:`FileCategory.EXCLUDE`
+    """
+
+    _term = u"trusted"
+
+    INCLUDE = True
```

Comment:
Does this actually work? In the UI, it is the strings "TRUE" and "FALSE" that gets sent in the request

Reply:
I noted that this should be a boolean filter type like we have for `Shared`.

Reply:
Changed